core/src/test/scala/integration/kafka/api/BaseAsyncConsumerTest.scala

```diff
@@ -144,4 +144,24 @@ class BaseAsyncConsumerTest extends AbstractConsumerTest {
     consumeAndVerifyRecords(consumer = consumer, numRecords, startingOffset = 10,
       startingTimestamp = startingTimestamp)
   }
+
+  @Test
+  def testConsumeFromCommittedOffsets(): Unit = {
```

Comment:
This is not consuming from the committed offsets, only fetching them, so maybe rename to testFetchCommittedOffsets

Reply:
thanks done!